index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Tic Tac Toe</title>
+  <link rel="stylesheet" href="style.css" type="text/css" media="screen"/>
+</head>
+
+<body>
+
+  <div class="tic-tac-toe">
+
+    <h1>Tic Tac Toe</h1>
+
+    	<div class="line">
+        <li class="button" id="one"></li>
+        <li class="button" id="two"></li>
+        <li class="button" id="three"></li>
+      </div>
+      <div class="line">
+        <li class="button" id="four"></li>
+        <li class="button" id="five"></li>
+        <li class="button" id="six"></li>
+  	  </div>
+      <div class="line">
+        <li class="button" id="seven"></li>
+        <li class="button" id="eight"></li>
+        <li class="button" id="nine"></li>
+  	  </div>
+
+      <!-- Clear Game Button-->
+
+      <div id="clear">Clear Game</div>
```

Comment:
Not a major issue, but just a heads up that it's customary to use a `<button>` element for any case where clicking it causes a major change (such as clearing a form, or in this case the board). It's mostly for accessibility reasons (screen reader software can better communicate that to the user).